csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedCall.qll

```diff
@@ -10,7 +10,7 @@ private import semmle.code.csharp.ir.internal.IRCSharpLanguage as Language
 
 /**
  * The IR translation of a call to a function. The function can be a normal function
- * (ie. `MethodCall`) or a constructor call (ie. `ObjectCreation`). Notice that the
+ * (eg. `MethodCall`) or a constructor call (eg. `ObjectCreation`). Notice that the
```

Comment:
Why this change? Also, in general, I believe it is `i.e.` and `e.g.`, and not `ie.` and `eg.`.

Reply:
This was part of the PR that got merged, probably wasn't picked up in the diff. Anyways, modified it to `i.e.` and `e.g.`.